release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -709,6 +709,13 @@ AWS Route53 endpoint configuration is now supported on the Ingress Operator.
 
 See xref:../installing/installing_aws/installing-aws-account.html#nw-endpoint-route53_installing-aws-account[Ingress Operator endpoint configuration for AWS Route53] for more information.
 
+[id="ocp-4-6-ingress-controller-unique-id-header"]
+==== Ingress Controller unique-id configuration
+
+Configuring an Ingress Controller to inject an HTTP header with a uniquely defined request ID is now supported. This can be used to trace cluster traffic.
```

Comment:
Lets denote the z stream / errata that provided this functionality with 4.6

Reply:
Ingress specification and uniqueID are in 4.7+ (https://docs.openshift.com/container-platform/4.7/rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.html#specification).